deepfence_frontend/apps/dashboard/src/features/vulnerabilities/pages/RuntimeBom.tsx

```diff
@@ -210,27 +207,36 @@ const RuntimeBom = () => {
 
   return (
     <div>
-      <div className="flex px-2 items-center w-full shadow bg-white dark:bg-gray-800 h-10">
-        <Breadcrumb separator={<HiChevronRight />} transparent>
-          <BreadcrumbLink>
-            <DFLink to={'/vulnerability'}>Vulnerabilities</DFLink>
+      {/* do we need shadow here? */}
+      <div className="flex pl-6 px-2 py-2.5 items-center w-full shadow bg-white dark:bg-bg-breadcrumb-bar">
+        <Breadcrumb>
+          <BreadcrumbLink
+            icon={
+              <div className="mr-2">
+                <VulnerabilityIcon />
```

Comment:
there is already a vulnerability icon, please use that instead of adding new svg.

Reply:
Also check how icons are used in other screens.